### PR TITLE
[chore] Format go mod files

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

**Review by codeboten — APPROVED** [review]

File: exporter/clickhouseexporter/go.mod [hunk 1]
```diff
@@ -10,16 +10,9 @@ require (
...
-	github.com/ClickHouse/clickhouse-go/v2 v2.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.63.0
-	github.com/stretchr/testify v1.8.1
-	go.opentelemetry.io/collector/pdata v0.63.2-0.20221108014805-0b08d5316426
-	go.opentelemetry.io/collector/semconv v0.63.2-0.20221108014805-0b08d5316426
-)
-
 require (
 	github.com/ClickHouse/ch-go v0.47.3 // indirect
+	github.com/ClickHouse/clickhouse-go/v2 v2.3.0
```
[the remark this answers is not in the document]
Comment (bogdandrutu) [reply exporter/clickhouseexporter/go.mod:15]: see the slack discussion

[the remark this answers is not in the document]
Comment (bogdandrutu) [reply exporter/clickhouseexporter/go.mod:15]: FYI: That is what I am trying to get... but apparently in this file there was another extra require first..

**Review by dmitryax — APPROVED** [review]
LGTM

**Review by TylerHelmuth — APPROVED** [review]
@bogdandrutu looks like some CI is failing

**Pull request merged**